guide/03-the-gis/accessing-and-managing-users.ipynb

```diff
@@ -1 +1,1457 @@
ground.esri.com/portal/home/js/jsapi/esri/css/images/item_type_icons/mapimages16.png' style=\"vertical-align:middle;\">Map Image Layer by tiles_publisher\n", "                        <br/>Last Modified: December 13, 2016\n", "                        <br/>0 comments, 0 views\n", "                    </div>\n", "                </div>\n", "                "], "text/plain": ["<Item title:\"Transport_tiles\" type:Map Service owner:tiles_publisher>"]}, "execution_count": 121, "metadata": {}, "output_type": "execute_result"}], "source": ["# reassign Transport_tiles to publisher1\n", "transport_tiles_item = tiles_pub_user_items[2]\n", "transport_tiles_item"]}, {"cell_type": "code", "execution_count": 122, "metadata": {}, "outputs": [{"data": {"text/plain": ["True"]}, "execution_count": 122, "metadata": {}, "output_type": "execute_result"}], "source": ["# the reassign_to() method accepts user name as a string. We can also specify a destination folder name\n", "transport_tiles_item.reassign_to(target_owner = 'publisher1', target_folder= 'f1_english')"]}, {"cell_type": "code", "execution_count": 123, "metadata": {}, "outputs": [{"data": {"text/plain": ["True"]}, "execution_count": 123, "metadata": {}, "output_type": "execute_result"}], "source": ["# now let us get rid of redundant ocean tiles items\n", "tiles_pub_user_items[1].delete()"]}, {"cell_type": "code", "execution_count": 124, "metadata": {}, "outputs": [{"data": {"text/plain": ["True"]}, "execution_count": 124, "metadata": {}, "output_type": "execute_result"}], "source": ["tiles_pub_user_items[-1].delete()  # an index of -1 in a list refers to the last item"]}, {"cell_type": "markdown", "metadata": {}, "source": ["Now we are left with a few more items which should all go to user `arcgis_python_api`. We can either call `reassign_to()` method of the `User` object or call the `delete()` method of the `User` object and pass this information to the `reassign_to` parameter. Let's do that:"]}, {"cell_type": "code", "execution_count": 125, "metadata": {}, "outputs": [{"data": {"text/plain": ["True"]}, "execution_count": 125, "metadata": {}, "output_type": "execute_result"}], "source": ["tiles_pub_user.delete(reassign_to='arcgis_python_api')"]}, {"cell_type": "markdown", "metadata": {}, "source": ["Thus, we have successfully deleted a user after taking care of that user's content."]}], "metadata": {"kernelspec": {"display_name": "Python 3", "language": "python", "name": "python3"}, "language_info": {"codemirror_mode": {"name": "ipython", "version": 3}, "file_extension": ".py", "mimetype": "text/x-python", "name": "python", "nbconvert_exporter": "python", "pygments_lexer": "ipython3", "version": "3.8.2"}, "toc": {"base_numbering": 1, "nav_menu": {}, "number_sections": true, "sideBar": true, "skip_h1_title": false, "title_cell": "Table of Contents", "title_sidebar": "Contents", "toc_cell": true, "toc_position": {}, "toc_section_display": true, "toc_window_display": true}}, "nbformat": 4, "nbformat_minor": 2}
```

Comment:
<p>"A set of <em>privileges</em> is assigned to a <em>role</em>, and then based upon those privileges a role will be compatible with certain <em>user</em>types_"</p><ul><li>is "usertypes" here actually "user_types" or "usertypes"?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1612/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
building the website locally looks like this. The source notebook text is `_user_type_` to italicize it:

![image](https://github.com/Esri/arcgis-python-api/assets/1399179/4e89fbb9-0a3f-40de-a433-8266f306c254)

Reply:
<p>This looks correct when building the Developer Website locally. The raw text to italicize a word is to surround it in underscores, so the raw text looks like _user_type_ - on the local build of the website it looks correct.</p>